torchdynamo/optimizations/backends.py

```diff
@@ -452,6 +452,16 @@ def run(*new_inputs):
     return run
 
 
+@create_backend
+def aot_autograd(subgraph, **kwargs):
+    from functorch.compile import aot_function, memory_efficient_fusion
```

Comment:
I kept the imports inside for now to avoid making functorch a dependency.

Reply:
That's perfect